src/bluetooth/TemperatureLogManager.test.js

```diff
@@ -96,6 +96,7 @@ describe('DownloadManager: createLogs', () => {
     // a log for anytime past 600, so we assume any excess logs are passed records which have been
     // saved previously and just save the two 'most recent'.
     const sensor = { id: 'a', logInterval: 300 };
+    const MILLISECONDS_PER_SECOND = 1000;
```

Comment:
or `import { MILLISECONDS_PER_SECOND } from '../database/utilities/constants';` at the top of the file, with an `export` in the constants.js?

Reply:
Hm, yeah I think I went over the top with trying to not include dependencies, constants should be pretty safe haha. Also found `src/utilities/constant` which has a MILLISECONDS constant so I used that one 61d41166